app/src/main/java/com/framgia/music/MainActivity.java

```
@@ -42,5 +93,154 @@ public boolean onOptionsItemSelected(MenuItem item) {
+            }
+        });
+    }
+
+    @Override
+    public void onClick(View view) {
+        setSeekBar();
+        switch (view.getId()) {
+            case R.id.btnPlay:
+                onPlayMusic();
+                break;
+            case R.id.btnNext:
+                onNextSongs();
+                break;
+            case R.id.btnPre:
+                onPeriousSongs();
+                break;
+        }
+    }
+
+    private void onPeriousSongs() {
+        mPos--;
+        if (mPos < 0) {
+            mPos = arraySong.size() - 1;
+        }
+        if (mMediaPlayer.isPlaying()) {
+            mMediaPlayer.stop();
+        }
+        initMedia();
+        mMediaPlayer.start();
+        mBtnPlay.setImageResource(R.drawable.pause);
+        setTimeEnd();
+        updateTimeSong();
+    }
+
+    private void onNextSongs() {
+        mPos++;
+        if (mPos > arraySong.size() - 1) {
+            mPos = 0;
+        }
+        if (mMediaPlayer.isPlaying()) {
+            mMediaPlayer.stop();
+        }
+        initMedia();
+        mMediaPlayer.start();
+        mBtnPlay.setImageResource(R.drawable.pause);
+        setTimeEnd();
+        updateTimeSong();
+    }
+
+    private void onPlayMusic() {
+        if (mMediaPlayer.isPlaying()) {
+            mMediaPlayer.pause();
+            mBtnPlay.setImageResource(R.drawable.play);
+        } else {
+            initMedia();
+            mImageView.startAnimation(mAnimation);
+            mMediaPlayer.start();
+            mBtnPlay.setImageResource(R.drawable.pause);
+            setTimeEnd();
+            updateTimeSong();
+        }
+    }
+
+    private void initMedia() {
+        mMediaPlayer =
+                MediaPlayer.create(MainActivity.this, Uri.parse(arraySong.get(mPos).getPath()));
+        txtNameSong.setText(arraySong.get(mPos).getName());
+    }
+
+    private void setTimeEnd() {
+        SimpleDateFormat formatHout = new SimpleDateFormat("mm:ss");
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.